tests/unittests/test_date_out_of_range.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+from unittest import mock
+from tap_jira.streams import Stream
+
+class TestOutOfRangeDate(unittest.TestCase):
+    @mock.patch("tap_jira.streams.singer.utils.now", return_value="2022-05-23T09:16:11.356670Z")
+    @mock.patch("tap_jira.streams.singer.write_record")
+    @mock.patch("tap_jira.streams.Context")
+    @mock.patch("tap_jira.streams.metadata")
+    def test_out_of_range_date(self, mock_metadata, mock_Context, mock_write_record, mock_now):
+        """
+        Verify that the tap successfully skips record for out of range date value.
+        """
+        mock_records =[
+            {"updated": "2017000-09-05T19:51:03.159Z", "id": "1","id2": "11"},  # year out of range
```

Comment:
Have we take any sample of `month must be in 1..12`?

Reply:
Yes, verified the same. Please see [L17](https://github.com/singer-io/tap-jira/pull/87/files#diff-38c5ac42c3c3fcae142935dae6ed190e2647fea82e6808ebc0ced96055a3387bR17) for reference